README_validations.md

```diff
@@ -0,0 +1,129 @@
+
+If working with XML you will need an XML *schema* (or functional equivalent) to validate your OSCAL. If working with JSON or YAML, you will need a JSON Schema. Normative schemas suitable for validating the various OSCAL models are available on the [project web site]().
+
+## What is my model?
+
+OSCAL comes in several different forms, depending on the data set. At time of writing we have the following models. Each will come with its own schema.
+
+| Model | XML Schema | JSON Schema |
+| -------- | -------- | -------- |
+| Catalog | LINK     | LINK     |
+| Profile | LINK     | LINK     |
+| System security plan (SSP) | LINK     | LINK     |
+| Component definition | LINK     | LINK     |
+| Assessment Plan | LINK     | LINK     |
+| Assessment Results | LINK     | LINK     |
+| Plan of Action & Milestones (POAM) | LINK     | LINK     |
+
+Once you have the model and the schema to validate it with, you are ready.
+
+## OSCAL "all" schema
+
+Additionally, for both XML and JSON we have an "all" schema, which can validate any kind of OSCAL, distinguishing them internally. It is larger and more complex than the model-specific schemas and may not be as efficient or useful; but it can be convenient for certain use cases.
+
+## How do I apply a schema to a document to determine validity?
+
+Schemas are built using standard schema languages, in order that commodity tools can all apply the same schemas to the same documents, and give the same results.
+
+For OSCAL XML Schemas, this syntax is XSD (XML Schema Definition). For JSON Schemas, it is JSON Schema v7. Free and open-source implementations are available for these; they are also built in to many common application frameworks and dev environments.
+
+Each tool will have its own way to provide for validation. Bindings or configurations to link documents to schemas can be maintained either inside or outside the documents themselves; many tools offer several ways to do this.
+
+Using a simple command-line tool such as [XML Lint](https://), however, validation against an XML Schema in XSD syntax can be done in a single line.
+
+```
+$ xmllint oscal.xml --schema oscal-schema.xsd --noout
+
+```
+
+(where `oscal-schema.xsd` is the appropriate schema.)
+
+Similarly, using AJV, validate a JSON document like this:
+
+```
+$ line to validate
+```
+
+## What does success look like?
+
+If you validate an OSCAL catalog with its schema (a local copy) as above (using XML Lint), it will return results to STDOUT like so:
+
+```
+test-profile.xml validates
+```
+
+In other words, no data is returned other than the statement that validation has been performed successfully.
+
+(If you remove the `--nonout` flag from the command line, a copy of the input will be returned. This is because the utility is designed for a pipeline architecture, whose outputs will be a functio of its inputs.)
```

Comment:
```suggestion
If you remove the `--nonout` flag from the command line, a copy of the input will be returned. This is because the utility is designed for a pipeline architecture, whose outputs will be provided as the next tool's inputs.
```